crates/fluvio-cli-common/src/version_cmd.rs

```diff
@@ -0,0 +1,86 @@
+use anyhow::Result;
+use clap::Args;
+use current_platform::CURRENT_PLATFORM;
+use crate::FLUVIO_RELEASE_CHANNEL;
+use sha2::{Digest, Sha256};
+
+const VERSION: &str = include_str!("../../../VERSION");
+
+/// Display version information
+#[derive(Debug, Args)]
+pub struct BasicVersionCmd;
+
+impl BasicVersionCmd {
+    /// Display basic information about the current fluvio installation
+    ///
+    /// The following information is displayed:
+    /// - Release channel, if available;
+    /// - CLI version;
+    /// - Platform arch;
+    /// - CLI SHA256, if available;
+    /// - channel frontend SHA256, if available;
+    /// - Git hash, if available;
+    /// - OS details, if available;
+    pub fn process(self) -> Result<()> {
```

Comment:
done, I chose to receive a name to display as an arg to "process" instead of reading the bin name to have a consistent display even if the user renames the binary - let me know if you think is better otherwise

Reply:
Feels a little weird, but it's reasonable for the consistency. I don't have an alternative to suggest